WrightTools/data/_spcm.py

```diff
@@ -37,9 +38,6 @@ def from_spcm(filepath, name=None, *, delimiter=",", format=None, parent=None, v
         Default is None.
     delimiter : string (optional)
         The string used to separate values. Default is ','.
-    format : {'ascii'} (optional)
-        Force file to be interpreted as a specific format. Default is None
-        (autorecognized).
```

Comment:
Was format used?, I didn't see it, but just checking to be sure

Reply:
format is not used